Tiny Cleanup - Removed duplicate class field definition's

Sai-Suraj-27 opened the pull request:
# What does this PR do?
Tiny cleanup. Used [Ruff-PIE794](https://docs.astral.sh/ruff/rules/duplicate-class-field-definition/). Will be happy to add the rule to [existing rules](https://github.com/huggingface/transformers/blob/e4930d6bde027ed21fd3dd91dbbfdc95e0221dd4/pyproject.toml#L36)


## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?


## Who can review?
@ydshieh @Rocketknight1

Sai-Suraj-27 committed:
Removed duplicate-class-field-definition
's using RUFF PIE794

Sai-Suraj-27 committed:
Merge branch 'main' into remove_duplicate_defs

Sai-Suraj-27 committed:
Removed duplicate-class-field-definition
's using RUFF PIE794

Sai-Suraj-27 committed:
Merge branch 'remove_duplicate_defs' of github.com-personal:Sai-Suraj-27/transformers into remove_duplicate_defs

Sai-Suraj-27 committed:
Ruff format.

Rocketknight1 reviewed (approved):
Yes, good fix! I also think this would be a nice rule to add to the ruff rules, cc @arthurzucker @cyrilvallez if you're okay with it.

Cyrilvallez commented on src/transformers/pipelines/text_to_audio.py:
```diff
@@ -84,7 +84,6 @@ class TextToAudioPipeline(Pipeline):
     _load_processor = True
 
     _pipeline_calls_generate = True
-    _load_processor = False
```
The wrong value has been kept here - only False should remain (the last value that was set)

Cyrilvallez reviewed (commented):
LGTM but the wrong one was kept in a file

Sai-Suraj-27 committed:
s
:wqerge branch 'main' of https://github.com/huggingface/transformers into remove_duplicate_defs

Sai-Suraj-27 committed:
Removed duplicate-class-field-definition

Sai-Suraj-27 committed:
Added New ruff rule to detect duplicate class field defs

Cyrilvallez committed:
remove comment

Cyrilvallez committed:
order

Cyrilvallez reviewed (approved):
LGTM, thanks for cleaning!
